packages/calypso-e2e/src/lib/pages/domains-page.ts

```diff
@@ -0,0 +1,92 @@
+import { Page } from 'playwright';
+
+const selectors = {
+	// Cart popover
+	cartButton: '.popover-cart button',
+	popOver: '.popover',
+	removeItemButton: 'button[aria-label="Remove item"]',
+	popOverCartPlaceholder: '.cart-item__loading-placeholder',
+
+	searchForDomainButton: `a:text-matches("(s|S)earch", "i")`,
```

Comment:
nit: the upper case should not be needed with the case-insensitive flag

```suggestion
	searchForDomainButton: `a:text-matches("search", "i")`,
```

...or the other way round:

```suggestion
	searchForDomainButton: `a:text-matches("(s|S)earch")`,
```

Reply:
Thanks!